modules/dcache/src/main/java/diskCacheV111/poolManager/RequestContainerV5.java

```diff
@@ -1396,7 +1407,7 @@ private void stateEngine( Object inputObject ) {
                         return ;
                     }
 
-                    if( ( rc = askIfAvailable() ) == RT_FOUND ){
+                    if( ( rc = askIfAvailable() ) == ReturnCode.FOUND.value ){
```

Comment:
Sure, but then you use enums just as a decorators around integers. If **askIfAvailable** will return **11**,  your code will compile and probably work. Nevertheless, **11** is not a legal value. On the other hand, if it returns enum ReturnCode (which is btw not a great name) then it can return only values defined by the enum. This makes your code more robust.